README.md

```diff
@@ -8,8 +8,7 @@
 
 # Jira link
 
-[Tracker - Kanban board](https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2)
-
+![Tracker - Kanban board](https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2)
```

Comment:
What is the use of "!" the exclamation point?

Reply:
It's a mistake, sorry I'll remove it for this one.